string id
---
float32 reward
float32[] actions
